src/components/autocomplete/AutocompleteConnected.tsx

```diff
@@ -0,0 +1,173 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {keyCode} from '../../utils/InputUtils';
+import {IDispatch, ReduxConnect} from '../../utils/ReduxUtils';
+import {IItemBoxProps} from '../itemBox/ItemBox';
+import {ListBoxConnected} from '../listBox/ListBoxConnected';
+import {addAutocomplete, removeAutocomplete, setAutocompleteActive, setAutocompleteValue, toggleAutocomplete} from './AutocompleteActions';
+import {IAutocompleteState} from './AutocompleteReducers';
+
+const defaultMatchFilter = (filterValue: string, item: IItemBoxProps) => {
+    if (filterValue === '') {
+        return true;
+    }
+
+    const regex = new RegExp(filterValue, 'gi');

```

Comment:
do you really need to match everywhere at this level or just to know it matches would do? if the latter is ok you could remove the `g`.

Reply:
This is handled by PartialStringMatch

Reply:
Yeah I only need to know if there's a match, the value of the match is useless to me